__tests__/app.test.js

```diff
@@ -47,4 +47,39 @@ describe('/api/topics', () => {
           });
         });
     });
-});
\ No newline at end of file
+});
+
+describe('/api/articles/:article_id', () => {
+    test('GET:200 sends an object containing the article related to the id to the client', () => {
+        return request(app)
+      .get('/api/articles/1')
+      .expect(200)
+      .then((response) => {
+        expect(response.body.article.title).toBe("Living in the shadow of a great man");
+        expect(response.body.article.topic).toBe("mitch");
+        expect(response.body.article.author).toBe( "butter_bridge");
+        expect(response.body.article.body).toBe( "I find this existence challenging");
+        expect(response.body.article).toHaveProperty("created_at");
+        expect(response.body.article.votes).toBe(100);
+        expect(response.body.article.article_img_url).toBe("https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700");
+      });
```

Comment:
Great test for your article object. You could also use a matcher like `expect.objectContaining` or `.toMatchObject` to make your test flexible 🙂